include/dlr_common.h

```diff
@@ -43,20 +43,20 @@ typedef struct {
   std::string metadata;
 } ModelPath;
 
-void ListDir(const std::string& dirname, std::vector<std::string>& paths);
+void ListDir(const std::string &dirname, std::vector<std::string> &paths);
```

Comment:
FYI: having the `*` or `&` adjadcent to type or variable name are both acceptable: https://google.github.io/styleguide/cppguide.html#Pointer_and_Reference_Expressions

We don't have to change all of these. 
It seems like clang-format moved these automatically though?

Reply:
Yeah!